src/proxy.c

```diff
@@ -1383,10 +1384,7 @@ cc_proxy_run_hyper_new_container (struct cc_oci_config *config,
 	json_object_set_string_member (process, "workdir",
 			config->oci.process.cwd);
 
-	/* Hyperstart is not able to handle uid 0. This is a bug.
-	 * Simply dont sent the uid for now as the container
-	 * runs as root by default until the bug is resolved.
-	 */
+	// Hyperstart is not able to handle uid 0. This is a bug.
```

Comment:
For consistency -- and to keep the diff minimal -- please can you use the existing C-style comment.